src/ostorlab/runtimes/local/runtime.py

```diff
@@ -658,7 +659,22 @@ def list_vulnz(
                         )
                     )
 
-                vulnerabilities = query.order_by(models.Vulnerability.title).all()
+                if order_by == "risk_rating":
+                    case_ordering = case(
+                        [
+                            (models.Vulnerability.risk_rating == rating, order)
+                            for rating, order in risk_rating.RATINGS_ORDER.items()
+                        ],
+                        else_=len(risk_rating.RATINGS_ORDER),
+                    )
+                    vulnerabilities = query.order_by(
+                        case_ordering, models.Vulnerability.title
+                    ).all()
+                elif order_by == "title":
+                    vulnerabilities = query.order_by(models.Vulnerability.title).all()
+                elif order_by == "id":
+                    vulnerabilities = query.order_by(models.Vulnerability.id).all()
```

Comment:
Else a warning would be nice.

Reply:
it will never be one other than those 3 !! this is handled by click.Choice already